Sources/Genything/Random/RandomSource.swift

```diff
@@ -39,7 +32,18 @@ public class RandomSource {
 // MARK: Convenience RandomSource Creators
 
 extension RandomSource {
-    /// Returns: An independent `RandomSource` initialized from a nondeterministic seed
+    /// Returns: An new, independent `RandomSource` initialized by default with a common seed for all Genything users
+    public static func predetermined(seed: UInt64 = 2022) -> RandomSource {
```

Comment:
Nice Addition! These methods will help users access the deterministic functionality of the library in a more easily understood manner.